src/Header.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './pokeStyles.css'
+
+class Header extends Component {
+    state = {  }
+    render() { 
+        return ( <Link to={`/pokemon`}>
```

Comment:
Your header should contain links to Home and the Pokemon search page. You also want to have active styling on the links.